ComputeDataDrivenCrossSection.py

```diff
@@ -23,17 +19,48 @@
 parser.add_argument('--system', metavar='text', default='pp', help='collision system (pp, pPb, PbPb)')
 parser.add_argument('--energy', metavar='text', default='5.02', help='energy (5.02)')
 parser.add_argument('--centrality', metavar='text', default='010', help='centrality (010, 3050)')
-parser.add_argument("--prompt", action='store_true', help='flag to compute prompt cross section', default=False)
-parser.add_argument("--FD", action='store_true', help='flag to compute FD cross section', default=False)
-parser.add_argument("--Dplus", action='store_true', help='flag to compute D+ cross section', default=False)
-parser.add_argument("--Ds", action='store_true', help='flag to compute Ds cross section', default=False)
+
+groupComponent = parser.add_mutually_exclusive_group(required=True)
+groupComponent.add_argument("--prompt", action='store_true', help='flag to compute prompt cross section', default=False)
+groupComponent.add_argument("--FD", action='store_true', help='flag to compute FD cross section', default=False)
+
+groupParticle = parser.add_mutually_exclusive_group(required=True)
+groupParticle.add_argument("--Dplus", action='store_true', help='flag to compute D+ cross section', default=False)
+groupParticle.add_argument("--Ds", action='store_true', help='flag to compute Ds cross section', default=False)
+groupParticle.add_argument("--Lc", action='store_true', help='flag to compute Lc cross section', default=False)
+
 parser.add_argument("--batch", action='store_true', help='suppress video output', default=False)
+
+groupCorrelation = parser.add_mutually_exclusive_group()
+groupCorrelation.add_argument("--corr", action='store_true',
+    help='assume raw yield and (prompt/feed-down) fraction uncertanties as correlated', default=True)
+groupCorrelation.add_argument("--uncorr", action='store_true',
+    help='assume raw yield and (prompt/feed-down) fraction uncertanties as uncorrelated', default=False)
+groupCorrelation.add_argument("--anticorr", action='store_true',
+    help='assume raw yield and (prompt/feed-down) fraction uncertanties as anti correlated', default=False)
 args = parser.parse_args()
 
 # Define arguments (systematic uncertainties from AliHFSystErr)
+from ROOT import TFile, TCanvas, TLegend, TGraphErrors, gROOT  # pylint: disable=import-error,no-name-in-module
+from ROOT import AliHFSystErr  # pylint: disable=import-error,no-name-in-module
+from utils.AnalysisUtils import ComputeCrossSection, GetPromptFDFractionCutSet
+from utils.StyleFormatter import SetGlobalStyle, SetObjectStyle, GetROOTColor
+
```

Comment:
Keep all imports at the beginning of the file

Reply:
I moved the import of the root packages after the argparse section because otherwise, when you run python script.py -h, you get the help message from root and not the one of the script. If we're not interested in keeping the help message of the script I can move this part before.

Reply:
This is funny behaviour! In any case, I think we do not need the help message and I would keep all the imports together.